### DISCUSSION TITLE
MQTT connection

### DISCUSSION THREAD

**Discussion by sico33** [klaasnicolaas/home-assistant-glow#224] [General]
Hello, I wanted to know if there is any way to connect the glow through mqtt?

Thanks

**Comment by klaasnicolaas** [comment]
Why would you want to do that?

**Reply by sico33** [reply]
Por que quiero conectar un esphome a HA sobre internet en un medidor remoto

**Reply by sico33** [reply]
I managed to connect it through MQTT to the broker, but I don't know how to disable the Api

19:56:14 | [I] | [ota:109] | Boot seems successful, resetting boot loop counter.
-- | -- | -- | --
19:57:58 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -60.00000 dBm with 0 decimals of accuracy
19:59:58 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -66.00000 dBm with 0 decimals of accuracy
20:01:22 | [E] | [api:127] | No client connected to API. Rebooting...
20:03:03 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -64.00000 dBm with 0 decimals of accuracy
20:05:03 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -62.00000 dBm with 0 decimals of accuracy
20:07:03 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -55.00000 dBm with 0 decimals of accuracy
20:09:03 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -65.00000 dBm with 0 decimals of accuracy
20:11:03 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -62.00000 dBm with 0 decimals of accuracy
20:11:22 | [I] | [ota:109] | Boot seems successful, resetting boot loop counter.
20:13:03 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -64.00000 dBm with 0 decimals of accuracy
20:15:03 | [D] | [sensor:126] | 'House - WiFi Signal': Sending state -62.00000 dBm with 0 decimals of accuracy
20:16:30 | [E] | [api:127] | No client connected to API. Rebooting...


</body>
</html>

**Reply by sico33** [reply]
INFO Reading configuration /config/esphome/lachota.yaml...
Failed config

dashboard_import: [source /config/esphome/lachota.yaml:23]
  
  Component dashboard_import requires component api.
  package_import_url: github://klaasnicolaas/home-assistant-glow/home_assistant_glow.yaml
time.homeassistant: [source /config/esphome/lachota.yaml:189]
  
  Component time.homeassistant requires component api.
  platform: homeassistant
  id: homeassistant_time

**Comment by sico33** [comment]
Hello [klaasnicolaas], is there any way to read the Irda? 

https://www.sumilec.com/wp-content/uploads/2018/07/elster4.jpg

**Reply by klaasnicolaas** [reply]
![image](https://user-images.githubusercontent.com/20448157/200851839-07e5abac-2554-4168-a611-c6042c0ca0cf.png)

Yes, it should have a pulse LED on it and above that is your pulse rate.
